compile error on linux

1) git clone latest code
2) cmake  &  make ,   The following compilation error occurs:

In file included from /var/zz/n2ngit2/n2n/include/n2n.h:162:0,
                 from /var/zz/n2ngit2/n2n/src/n2n.c:19:
/var/zz/n2ngit2/n2n/include/n2n_port_mapping.h:7:33: fatal error: miniupnpc/miniupnpc.h: No such file or directory
 #include <miniupnpc/miniupnpc.h>
                                 ^
compilation terminated.

--- a/include/n2n_port_mapping.h
+++ b/include/n2n_port_mapping.h
@@ -4,9 +4,17 @@
 #include <stdint.h>
 
 #ifdef HAVE_MINIUPNP
+#ifdef CMAKE_BUILD
+// CMAKE uses static linked lib as submodule which requires different includes than
+//       the dynamically linked, intalled library in case of plain make
+#include <miniupnpc.h>
+#include <upnpcommands.h>
+#include <upnperrors.h>
+#else
 #include <miniupnpc/miniupnpc.h>
 #include <miniupnpc/upnpcommands.h>
 #include <miniupnpc/upnperrors.h>
+#endif // CMAKE_BUILD
 #endif // HAVE_MINIUPNP
 
 

